src/plugins/v-focus/directives/focus-mask.ts

```diff
@@ -1,13 +1,35 @@
 import state from '../state';
 
+const bemBlock = 'v-focus-mask';
+
 /**
  * Adds or removes a focus class to the given element based on the current focus state.
  */
 function update(el: HTMLElement): void {
```

Comment:
question: Can we change the name of this function to `updated` and rename its calls on lines 46 and 50 in the file accordingly?

I`ve testet it, works fine.